roles/ocitysmap/templates/french-style.sh.j2

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+cd /srv/gis/styles
+
+#git clone https://github.com/cquest/osmfr-cartocss.git
```

Comment:
Is the git clone commented because it is done in ansible script ?

If so, a comment telling so would be nice.

Reply:
hmmm ....
you're right i'm going to review this file ...